### PR TITLE
Test: S-3 Create, Get & Delete Class

### PR DESCRIPTION
**Pull request by Pansysk75**
Adding tests for the following endpoints:
- GET /admin/class/{className}
- POST ​/admin​/class
- DELETE ​/admin​/class/{className}
- GET /user/{userName}/class/{className}

and for their corresponding service functions:
- getClassInfoAdmin
- createClass
- deleteClass
- getClassInfoUser

### CODE REVIEW

**Comment by Nick-Liou** [comment]
You should write a **description**

**Comment by Pansysk75** [comment]
> You should write a **description**

I don't remember asking you to review this code, but I never say no to more work, so here's your description I hope you enjoy it.

File: tests/S-3_Create_Get_Delete_Class.test.js
Comment (Nantina) [inline tests/S-3_Create_Get_Delete_Class.test.js:1]: amazing guys! All the tests that are included in the file are working as expected. 
What do you think about adding some more test cases? Like multiple requests, empty requests and requests with wrong or partial input?

Comment (deepblue597) [reply tests/S-3_Create_Get_Delete_Class.test.js:1]: There are already multiple and empty request tests. Can you explain a bit more about what are you thinking of?

Comment (Nantina) [reply tests/S-3_Create_Get_Delete_Class.test.js:1]: I am referring to the GET requests. POST and DELETE have them already implemented as you mentioned.

**Review by Nantina — APPROVED** [review]

**Pull request merged**